src/handlers/errorHandler.ts

```diff
@@ -0,0 +1,51 @@
+import * as vscode from 'vscode';
+import { CloseAction, ErrorAction, ErrorHandler, Message } from 'vscode-languageclient';
+
+export class ExtensionErrorHandler implements ErrorHandler {
+  constructor(private outputChannel: vscode.OutputChannel) {}
+  error(error: Error, message: Message | undefined, count: number | undefined): ErrorAction {
```

Comment:
```suggestion
  error(error: Error, message: Message | undefined, count: number | undefined) {
```
Fun thing with implementing interfaces is, that you have to type the function arguments, but you don't have to type the return type.